lib/factory.js

```diff
@@ -276,17 +276,17 @@ var arrayToJson = function(array) {
   const cols = array.shift();
   const rowToObject = function(row) {
     const obj = {};
-    for (var [k, v] of Array.from(_.zip(cols, row))) {
```

Comment:
I wonder, how does _ get defined here since there is no require('underscore') as there is in the other files in this commit?

Reply:
Oh, good catch! Looks like we are also injecting it as a script tag on the page, which then causes it to get added as a global. Not sure if plugins depend on that 🙈 This PR at least gets it out of the client bundle/build process.